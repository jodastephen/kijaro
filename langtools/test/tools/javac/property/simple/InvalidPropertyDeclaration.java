package property.simple;

/*
 * @test @(#)InvalidPropertyDeclaration.java 1.7 10/06/07
 * @bug 4228585
 * @summary support property syntax check
 * @author freddy33
 *
 * @library ../..
 * @compile/fail/ref=InvalidPropertyDeclaration.out -XDstdout -XDdiags=%b:%l:%_%m InvalidPropertyDeclaration.java
 */
public class InvalidPropertyDeclaration {

    public property String ok;
    public property error;
    public property int good;

}