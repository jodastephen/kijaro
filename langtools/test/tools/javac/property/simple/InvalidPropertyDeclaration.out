InvalidPropertyDeclaration.java:15: <identifier> expected
    public property error;
                    ^
1 error